File: app/Model/UserInfo.ts
//     Copyright (c) Microsoft Corporation.  All rights reserved.

/// <reference path="../references/index.d.ts" />

module DXLiquidIntel.App.Model {

    // Need to keep structure in sync with DashServer.ManagementAPI.Models.OperationState in the WebAPI
    export class UserInfo {
        public PersonnelNumber: number
        public UserPrincipalName: string
        public UntappdUserName: string
        public UntappdAccessToken: string
        public CheckinFacebook: boolean
        public CheckinTwitter: boolean
        public CheckinFoursquare: boolean
        public FullName: string
        public FirstName: string
        public LastName: string
        public IsAdmin: boolean
        public ThumbnailImageUri: string
    }
}
         

File: mobile/xamarin/SmartKegerator/SmartKegerator/Models/UserInfo.cs
using System;
namespace SmartKegerator.Models
{
    public class UserInfo
    {
        public int PersonnelNumber { get; set; }
        public string UserPrincipalName { get; set; }
        public string UntappdUserName { get; set; }
        public string UntappdAccessToken { get; set; }
        public bool CheckinFacebook { get; set; }
        public bool CheckinTwitter { get; set; }
        public bool CheckinFoursquare { get; set; }
        public string FullName { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public bool IsAdmin { get; set; }
        public string ThumbnailImageUri { get; set; }
    }
}
